packages/amplication-client/src/CommandPalette/CommandPalette.tsx

```diff
@@ -122,8 +122,9 @@ const CommandPalette = ({ trigger }: Props) => {
   const { data } = useQuery<TData>(SEARCH, {
     variables: { query, projectId: currentProject?.id },
   });
-  const commands = useMemo(
+    const commands = useMemo(
     () =>
+  // @ts-ignore
```

Comment:
I prefer to fix the root cause, and generaly we don't accept or push any additional `// @ts-ignore` statements, we see it as a bad practice.